docs/getting-started/install-crossplane.md

```diff
@@ -0,0 +1,83 @@
+---
+toc_hide: true
+---
+
+## Install Crossplane
+
+Crossplane installs into an existing Kubernetes cluster. 
+
+{{< hint type="tip" >}}
+If you don't have a Kubernetes cluster create one locally with [minikube](https://minikube.sigs.k8s.io/docs/start/) or [kind](https://kind.sigs.k8s.io/).
+{{< /hint >}}
+
+### Install the Crossplane Helm chart
+
+Helm enables Crossplane to install all its Kubernetes components through a _Helm Chart_.
+
+Enable the Crossplane Helm Chart repository:
+
+```shell
+helm repo add \
+crossplane-stable https://charts.crossplane.io/stable
+helm repo update
+```
+
+Helm supports a `dry-run` to see the changes Helm makes. Run the Helm dry-run to see all the Crossplane components.
+
+```shell
+helm install crossplane \
+crossplane-stable/crossplane \
+--dry-run --debug \
+--namespace crossplane-system \
+--create-namespace
+```
+
+Last, install the Crossplane components using `helm install`.
+
+```shell
+helm install crossplane \
+crossplane-stable/crossplane \
+--namespace crossplane-system \
+--create-namespace
+
+NAME: crossplane
+LAST DEPLOYED: Sun Oct 16 19:11:27 2022
+NAMESPACE: crossplane-system
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+Release: crossplane
+
+Chart Name: crossplane
+Chart Description: Crossplane is an open source Kubernetes add-on that enables platform teams to assemble infrastructure from multiple vendors, and expose higher level self-service APIs for application teams to consume.
+Chart Version: 1.9.1
+Chart Application Version: 1.9.1
+
+Kube Version: v1.25.2
+```
+
+Verify Crossplane installed with `kubectl get pods`.
+
+```shell
+kubectl get pods -n crossplane-system
+NAME                                      READY   STATUS    RESTARTS   AGE
+crossplane-d4cd8d784-ldcgb                1/1     Running   0          54s
+crossplane-rbac-manager-84769b574-6mw6f   1/1     Running   0          54s
```

Comment:
This is not valid Bourne shell.

Reply:
Same as before, this is just providing hints to the syntax highlighter.

Reply:
Reasoning has been added to the [contributing guide](https://github.com/crossplane/crossplane/pull/3396/files#diff-93426d953f03b733adbbb574ffd2717d0892773a464c0d301d1b6c9095d67b15R191). 

Resolving this conversation (unresolve if you're not satisfied with this)

Reply:
What do you think about keeping commands separate from output? This is something the Kubernetes docs recommend. Ideally they'd say why, but I'm guessing it's to make it clear which bits are a command you can copy/paste and which bits are output.

https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output

Reply:
@sftim brought up a similar point [here](https://github.com/crossplane/crossplane/pull/3396/files#discussion_r1004680374).

Copying the previous answer:
> I'm making a different editorial choice. It's easier to see the command and output in a single place. The main motivator for separating the command from output would be to simplify copy/paste. That will be address in the Crossplane docs in the near future with a copy button that allows the author to define which lines in the output the button copies.

Reply:
> That will be address in the Crossplane docs in the near future with a copy button that allows the author to define which lines in the output the button copies.

Will this also visually distinguish which parts are input from output?

It would be great to track these kinds of planned improvements publicly. Perhaps an issue on https://github.com/crossplane/crossplane.github.io?